client/navigation/Navigation.js

```diff
@@ -5,20 +5,21 @@ import { Login, AddActivity, Camera, MapComp, Dashboard, Product, LoadingScreen,
 
 const Drawer = createDrawerNavigator(
   {
-    dashboard: { screen: Dashboard },
+    Dashboard: { screen: Dashboard },
```

Comment:
why are we using uppercase here? please keep the upper/lower consistent. Here we are using camelCase for navigation terms.

Reply:
Yes, the file names are upper case and such...but we're stuck with that naming convention for now. In the examples I saw using navigation, they used the lowercase for the navigation key and uppercase for the view. Let's stick with that for now.